packages/access-api/test/helpers/ucanto-test-utils.js

```diff
@@ -38,13 +38,14 @@ export function createTesterFromContext(createContext, options) {
  * using a service-issued voucher/redeem invocation
  *
  * @param {Iterable<Resolvable<Ucanto.Principal>>} spaces
- * @param {Ucanto.Signer<Ucanto.DID>} issuer
- * @param {Ucanto.ConnectionView<Record<string, any>>} conn
+ * @param {object} options
+ * @param {Ucanto.Signer<Ucanto.DID>} options.service
```

Comment:
In the future I expect to have `spaceRegistrationInvocation` create a `provider/add` issued by an agent (not the 'service'), which is why I called it the more generic `issuer`. I can always change it back if/when we use non-service-issuers for this.